From 88e403cc40847a1dca76e50f4cba3fa5354526d9 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 23 Mar 2017 10:04:54 -0700
Subject: [PATCH 023/532] drm: xilinx: dp: End the comment on a separate line

commit dfa93ddb7d6f40aab4961071c1a0c063025f0d0e from
https://github.com/Xilinx/linux-xlnx.git

From checkpatch.pl,

WARNING: Block comments use a trailing */ on a separate line
+	* So, This loop should exit before 512 iterations */

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_dp.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_dp.c b/drivers/gpu/drm/xilinx/xilinx_drm_dp.c
index 7c86a9f..d554123 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_dp.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_dp.c
@@ -639,7 +639,8 @@ static int xilinx_drm_dp_link_train_cr(struct xilinx_drm_dp *dp)
 			  DP_TRAINING_PATTERN_1);
 
 	/* 256 loops should be maximum iterations for 4 lanes and 4 values.
-	 * So, This loop should exit before 512 iterations */
+	 * So, This loop should exit before 512 iterations
+	 */
 	for (max_tries = 0; max_tries < 512; max_tries++) {
 		ret = xilinx_drm_dp_update_vs_emph(dp);
 		if (ret)
-- 
2.7.4

